src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
     path: row.path,
     organizationId: row.org_id,
-    usersCount: parseInt(row.users_count)
+    usersCount: parseInt(row.users_count),
+    total: row.total
```

Comment:
just to check my decision with you: pg has a flag parseInt8 that generates Number out of Int8 (like in the COUNT case). [See a test sample](https://github.com/brianc/node-postgres/blob/f6c40b9331c90d794d5fcbb1d4ae2f28eabd4d42/test/integration/client/parse-int-8-tests.js#L14)
I converted manually for that case (reason: pg was instanced in 2 places in code and load test data) but we could enable the flag if you agree that no breaking changes are generated and pg instance(s) have the flag enabled.